Sources/String+Sweetness.swift

```diff
@@ -14,4 +50,14 @@ public extension String {
             return self
         }
     }
+
+    public func substring(with range: NSRange) -> Substring? {
+        guard let r = range.range(on: self as NSString) else { return nil }
```

Comment:
I know `range = range.range` is repetitive but please try not to use single letter variable names

Reply:
That's way more confusing, because one of them is a `Range<>` and the other an `NSRange`. Dammit, 🍎…